pkg/models/system_intake.go

```diff
@@ -35,51 +35,66 @@ const (
 	SystemIntakeStatusLCIDISSUED SystemIntakeStatus = "LCID_ISSUED"
 )
 
+// SystemIntakeRequestType represents the type of a system intake
+type SystemIntakeRequestType string
+
+const (
+	// SystemIntakeRequestTypeNEW captures enum value of "NEW"
```

Comment:
This const block can be combined with the previous (see models/business_case.go for an example)